webapi/src/main/scala/org/knora/webapi/responders/v1/ValuesResponderV1.scala

```diff
@@ -1939,11 +1946,13 @@ class ValuesResponderV1 extends ResponderV1 {
 
             // Generate a SPARQL update string.
             sparqlUpdate = queries.sparql.v1.txt.createLink(
+                resourceIndex = 0,
                 dataNamedGraph = dataNamedGraph,
                 triplestore = settings.triplestoreType,
                 resourceIri = resourceIri,
                 linkUpdate = sparqlTemplateLinkUpdate,
-                maybeComment = comment
+                maybeComment = comment,
+                checkObj = false
```

Comment:
Should this be `checkObj = true`? Since this method isn't used when creating multiple resources, we can assume that the target of the link already exists, right?